docs/guide/Titanium_SDK/Titanium_SDK_Guide/Atom_Package/Atom_Package_Release_Notes/Atom_Package_1.1.0_Release_Note.md

```diff
@@ -9,7 +9,7 @@ Atom Package 1.1.0 is a minor release that includes new features, improvements,
 
 ## New features
 
-* [ATOM-11](https://jira.appcelerator.org/browse/ATOM-11) - Snippets
+* [ATOM-11](https://jira-archive.titaniumsdk.com/ATOM-11) - Snippets
```

Comment:
We don't have ATOM tickets. Didn't even know there was an ATOM project in JIRA. Oops. Just leave the `ATOM-11` text and remove the link. Repeat for all ATOM-* links.